README.md

```diff
@@ -46,6 +46,11 @@ Launch the web service:
 cargo run --bin battleship-web-server --release
 ```
 
+Install trunk:
+```
+cargo install trunk
+```
+
```

Comment:
There's a link above for installing Yew: 
https://yew.rs/docs/getting-started/introduction